docs/layouts/shortcodes/mermaid.html

```diff
@@ -0,0 +1,22 @@
+{{ if not (.Page.Scratch.Get "mermaid") }}
+<!-- Include mermaid only first time -->
+<!--
+======
+This is a copy of https://github.com/alex-shpak/hugo-book/blob/master/layouts/shortcodes/mermaid.html
```

Comment:
I've moved all our custom shortcodes to the root to keep theme as close to the original as possible.